pypeit/scripts/ql_keck_mosfire.py

```diff
@@ -120,7 +120,7 @@ def run_pair(A_files, B_files, caliBrate, spectrograph, det, parset, show=False,
     # skymodel, objmodel, ivarmodel, outmask, sobjs, scaleimg, waveimg, tilts = redux.run(
     #     std_trace=std_trace, return_negative=True, show_peaks=show)
 
-    global_sky, sobjs_obj, skymask = redux.run_objfind(std_trace=std_trace, show_peaks=show)
+    global_sky, sobjs_obj, skymask = redux.run_objfind(std_trace=std_trace)
```

Comment:
Is there a reason why this has been disabled? Please do not remove the previous show_peaks functionality.